src/main/java/org/opentripplanner/updater/stoptime/TimetableSnapshotSource.java

```diff
@@ -272,46 +296,28 @@ public void applyTripUpdates(final Graph graph, final boolean fullDataset, final
     /**
      * Determine how the trip update should be handled.
      *
-     * @param tripUpdate trip update
+     * @param tripDescriptor trip descriptor
      * @return TripDescriptor.ScheduleRelationship indicating how the trip update should be handled
      */
-    private TripDescriptor.ScheduleRelationship determineTripScheduleRelationship(final TripUpdate tripUpdate) {
+    private TripDescriptor.ScheduleRelationship determineTripScheduleRelationship(final TripDescriptor tripDescriptor) {
         // Assume default value
         TripDescriptor.ScheduleRelationship tripScheduleRelationship = TripDescriptor.ScheduleRelationship.SCHEDULED;
 
         // If trip update contains schedule relationship, use it
-        if (tripUpdate.hasTrip() && tripUpdate.getTrip().hasScheduleRelationship()) {
-            tripScheduleRelationship = tripUpdate.getTrip().getScheduleRelationship();
-        }
-
-        if (tripScheduleRelationship.equals(TripDescriptor.ScheduleRelationship.SCHEDULED)) {
```

Comment:
It was used to check if a SCHEDULED trip update contained any skipped stops. If it did, this code changed the ScheduleRelationship to MODIFIED. It is no longer necessary because I added the skipped stop handling to the SCHEDULED trip updater handler.

Reply:
Yes.

I think I will take some time in the near future to re-enable the tests for this class as changes are quite risky at the moment.